src/plugin/negativeYear/index.js

```diff
@@ -0,0 +1,20 @@
+export default (_, c, dayjs) => {
+  const proto = c.prototype
+
+  const parseDate = (cfg) => {
+    const { date } = cfg
+    const newDate = new Date(date)
+    const fullYear = newDate.getFullYear()
+    if (typeof date === 'string' && date.indexOf(`-${fullYear}`) !== -1) {
+      return dayjs(newDate).subtract(fullYear * 2, 'year').toDate()
+    }
+    return newDate
```

Comment:
maybe we should return date not newDate, cause `cfg.date` is the untouched original one, just leave it to the current main parse logic

Reply:
@iamkun Updated